code/datums/elements/surgery_initiator.dm

```diff
@@ -0,0 +1,147 @@
+/// Allows an item to  be used to initiate surgeries.
```

Comment:
Put this in a module in the modular Skyrat folder, we shouldn't be adding modular files to non-modular folders.

Reply:
Also I don't even think it was ticked 💀